prompb/write/v2/types.proto

```diff
@@ -111,23 +113,43 @@ message Histogram {
   // Negative Buckets.
   repeated BucketSpan negative_spans =  8 [(gogoproto.nullable) = false];
   // Use either "negative_deltas" or "negative_counts", the former for
-  // regular histograms with integer counts, the latter for float
-  // histograms.
+  // regular exponential histograms with integer counts, the latter for exponential
```

Comment:
I don't think this is necessary.

Reply:
oh this is even wrong, sorry, I meant to do this on negative/positive spans. Should I?

Reply:
it's not wrong, but I think it's better to explain the usage on the bucket level (where it says Negative Buckets, Positive buckets) because we might be reusing these later for other schemas.